suite/src/benchmark.rs

```diff
@@ -20,56 +27,60 @@ pub fn get_cycle() -> u64 {
     ((counter_hi as u64) << 32u64) + counter_lo as u64
 }
 
+// TODO: implement general benchmarking function for sha265 module
+
```

Comment:
This TODO seems unnecessary now as you have already implemented a general function below?